pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -193,199 +221,87 @@ def get_commuter_data(read_data=dd.defaultDict['read_data'],
-                                checksum += val
-                                mat_commuter_migration[current_row,
-                                                       current_col] = val
-
-                    # take summed values of other REMAINING counties of a whole Bundesland
-                    # here, some counties of the Bundesland are stated explicitly and the rest is summed
-                    # the first or is for the case that the right first line of the incoming people directly
-                    # addresses one
-                    # the latter 'or's is used if no single county nor gov region of a federal state is stated
-                    # explicitly
-                    # although there are existent government regions in this federal state (i.e., the state itself is
-                    # not considered a governement region according to gov_list)
-                    elif ((str(commuter_migration_file.iloc[i][3]) == 'Übrige Regierungsbezirke (Bundesland)' and str(
-                            commuter_migration_file.iloc[i][4]).isdigit())
-                          or ((commuter_migration_file.iloc[i][2]).isdigit() and str(
-                              commuter_migration_file.iloc[i - 1][2]).startswith('nan'))
-                          or (len(str(commuter_migration_file.iloc[i][2])) == 2 and
-                              abs(float(commuter_migration_file.iloc[i][2]) - float(
-                                  commuter_migration_file.iloc[i - 1][2])) == 1)
-                          or (len(str(commuter_migration_file.iloc[i][2])) == 2 and
-                              abs(float(commuter_migration_file.iloc[i][2]) - float(
-                                  commuter_migration_file.iloc[i - 1][2])) == 2)):
-
-                        # auxiliary key of Bundesland (key translated to int starting at zero)
-                        dummy_key = int(commuter_migration_file.iloc[i][2]) - 1
-
-                        # sum population of all counties not explicitly migrated
-                        # from the current gov region migrated from
-                        dummy_pop_sum = 0
-                        for j in range(0, len(state_gov_table[dummy_key])):
-                            # over all government regions not explicitly stated
-                            gov_index = govkey2numlist[state_gov_table[dummy_key][j]]
-                            for k in range(0,
-                                           len(gov_county_table[gov_index])):
-                                # over all counties of the considered gov region
-                                if counties_migratedfrom[gov_index][k] < 1:
+    with progress_indicator.Spinner() as p:
+        for i in range(len(commuter_migration_files)):
+            p.set_message('Progress '+str(i+1)+'/16')
+            commuter_migration_file = commuter_migration_files[i]
+
+            counties_done = []  # counties considered as 'migration from'
```

Comment:
Update these comments. Hard to understand if someone is not familiar with our mobility naming.